client/src/Components/Header/AnonymousDiv.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+import { LogInput } from './AnonymousDivStyles';
+import { LogDiv } from './AnonymousDivStyles';
+import { LogButton } from './AnonymousDivStyles';
+
+export const AnonymousDiv = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
```

Comment:
Czy to jest nam potrzebne?
Wydaje mi sie że generalnie wszystkie dane będą pobierane po kliknięciu w button wyślij więc chyba nie potrzebujemy tego